File: nivel-01/ex-02/preco_carro.c
#include <stdio.h>
#include <assert.h>

//
// Retorna o custo final da fabricação de um carro
//
float custoFinal(float precoProducao) {
    float custoDistruidor = precoProducao * 28/100;
    float custoImposto = precoProducao * 45/100;
    return precoProducao + custoImposto + custoDistruidor;
}

//
// Teste
//
int main() {

    assert(17300.0f == custoFinal(10000));

    return 0;

}

File: nivel-01/ex-02/preco_carro.js
var assert = require('assert');

//
// Retorna o custo final da fabricação de um carro
//
var custoFinal = function(precoProducao) {
	var custoDistribuidor = precoProducao * 28/100
	var custoImposto = precoProducao * 45/100
	return precoProducao + custoImposto + custoDistribuidor
}

//
// Seu teste
//
try {
    assert.equal(17300, custoFinal(10000));
} catch(e) {
    console.log(e);
}